### DISCUSSION TITLE
RabbitMq Node statistics not available in the third node(RabbitMQ 3.9.8 Erlang 24.1)

### DISCUSSION THREAD

**Discussion by rajamadhu** [rabbitmq/rabbitmq-server#3675] [Other]
I have installed RabbitMQ in centos in node1(master) and node2(slave) is 3.8.2 Erlang 22.2 lower version and added new node 3 (slave) is RabbitMQ 3.9.8 Erlang 24.1,when i clustered node1 and node 2 is shows fine and but node 3 is i can able to join the cluster when i see the RabbitMQ Overview Node statistics not available in the node3.

when check the cluster status all three nodes are able to see and policy
i compared both version management plugin enabled.
enabled rabbitmq-plugins rabbitmq_management and restarted services.

node1 and node 2 when i check the management below screen shot.
 ![alt text](https://i.stack.imgur.com/rejnu.png)

when i check the 3rd node  UI management console below screen shot.
 ![alt text](https://i.stack.imgur.com/cMzxn.png)

kindly suggest if anything i missed.

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
A newly joined node will emit stats after a [period of time](https://rabbitmq.com/management.html#statistics-interval). Before that happens, it won't have any stats to report and the UI won't have any to display.

Different cluster nodes may have different set of plugins or settings, which can also affect what stats are collected or how often.

Finally, mixed version clusters (such as this 3.8.2 and 3.9.8 cluster) are not guaranteed to provide every feature or be compatible in every way. They don't have any major incompatibilities but 3.8 nodes obviously won't be able to interpret some 3.9-specific data, for example.
